mautic/mautic
# Merge 2.16.0 to 3.x

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Merging changes that were added between the release of 2.16.0-beta and 2.16.0. 
Plus `composer update` to resolve conflicts in composer.lock

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Make sure Mautic loads. The rest should be tested by individual PR testers.
